src/components/Tooltip/Tooltip.spec.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import { fireEvent, render, waitForDomChange } from '@testing-library/react';
+
+import Tooltip from './Tooltip';
+
+const getProps = (props = {}) => ({
+  className: props.className,
+  disableFocusListener: props.disableFocusListener,
+  disableMouseListener: props.disableMouseListener,
+  enterDelay: props.enterDelay !== undefined ? props.enterDelay : 0,
+  leaveDelay: props.leaveDelay !== undefined ? props.leaveDelay : 0,
+  onBlur: jest.fn(),
+  onClose: jest.fn(),
+  onFocus: jest.fn(),
+  onMouseEnter: jest.fn(),
+  onMouseLeave: jest.fn(),
+  onOpen: jest.fn(),
+  style: props.style,
+  testId: 'tip',
+  text: 'I am the tooltip'
+});
+
+const getTooltip = props => (
+  <Tooltip {...props}>
+    <div>Kid</div>
+  </Tooltip>
+);
+
+describe('<Tooltip />', () => {
+  describe('render', () => {
+    it('should render wrapper element', () => {
+      const props = getProps();
+
+      const { getByTestId } = render(getTooltip(props));
```

Comment:
Do you think the code could be more readable calling `getProps` inside of `getTooltip` directly? (Since we won't use the `props` variable anywhere else)

_Maybe_ something like this:
```javascript
render(getTooltip(getProps()))
```

Reply:
this makes sense in the tests where props isn't accessed again later! it's accessed in the callback tests, so those will have to stay put.

Reply:
updated in https://github.com/arturbien/React95/commit/82e7600b527fd3f9a89548b1231816ec50a3680c